core/src/com/po/fuck/model/Core.java

```diff
@@ -0,0 +1,47 @@
+package com.po.fuck.model;
+
+import static com.po.fuck.model.Constants.GAME_HEIGHT;
+import static com.po.fuck.model.Constants.GAME_WIDTH;
+
+import com.badlogic.gdx.math.Vector2;
+import com.po.fuck.model.collections.All;
+import com.po.fuck.model.collections.DrawableCollection;
+import com.po.fuck.model.enemies.BasicEnemy;
+import com.po.fuck.model.lifetime.Destructable;
+import com.po.fuck.model.lifetime.Managed;
+import com.po.fuck.model.lifetime.Manager;
+
+public class Core implements Updatable{
+    static {
+       forceInit(Destructable.class);
+       forceInit(All.class);
+    }
+    
+    public static Managed<Player> player;
```

Comment:
If it `Core` has non-static `initialize`, then player should not be static either.